_config.yml

```diff
@@ -5,6 +5,8 @@ relative_permalinks: false
 markdown: kramdown
 site_env: prod
 env:
+    hj:
+        hj: 'http://jeevesh.cadc.dao.nrc.gc.ca'
```

Comment:
Omit the `_config.yml` from the commits.

Reply:
ok